db/migrate/20220812142716_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] As per the given ERD diagram _user_ on the _post_ model is referenced as an _author_, similar to:
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

Reply:
Hi @rloterh, I have updated according to the ERD as you suggested. Thanks for the feedback.